.github/workflows/pull-request-ci.yaml

```
@@ -0,0 +1,125 @@
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: pnpm
+          cache-dependency-path: ./pnpm-lock.yaml
+      - run: pnpm install
+      - uses: actions/cache@v4
+        with:
+          path: node_modules/.cache/eslint
+          key: ${{ runner.os }}-eslint-${{ hashFiles('eslint.config.mjs') }}
+      - uses: reviewdog/action-eslint@v1
+        with:
+          fail_level: error
```

actionlint (1.7.4)

input "fail_level" is not defined in action "reviewdog/action-eslint@v1". available inputs are "eslint_flags", "fail_on_error", "filter_mode", "github_token", "level", "node_options", "reporter", "reviewdog_flags", "tool_name", "workdir"

(action)